CONTRIBUTING.md

```diff
@@ -14,7 +14,8 @@ Part of the GitHub Flow is submitting pull requests. See
 https://help.github.com/articles/using-pull-requests/ for a good overview of
 what pull requests are and how to use them. Internally, we use the _shared
 repository model_ for collaborative development, though we encourage external
-contributors to contribute through the _fork & pull model_.
+contributors to contribute through the _fork & pull model_ and assign a member
+of the core team to review the pull request.
```

Comment:
Yeah, one option may be to do that.

We could turn pull requests back on in our ZenHub board and treat them
similar to issues. It may get confusing when the pull request is in "In
Progress" or "Review" though.

On Thu, Aug 27, 2015 at 10:11 PM, Max Wofford <notifications@github.com>
wrote:

> In CONTRIBUTING.md
> <https://github.com/hackedu/hackedu/pull/104#discussion_r38171518>:
>
> > @@ -14,7 +14,8 @@ Part of the GitHub Flow is submitting pull requests. See
> >  https://help.github.com/articles/using-pull-requests/ for a good overview of
> >  what pull requests are and how to use them. Internally, we use the _shared
> >  repository model_ for collaborative development, though we encourage external
> > -contributors to contribute through the _fork & pull model_.
> > +contributors to contribute through the _fork & pull model_ and assign a member
> > +of the core team to review the pull request.
>
> Could we review PRs that aren't assigned at standups?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hackedu/hackedu/pull/104/files#r38171518>.
>



-- 
Zach Latta
hackEDU | zach@hackedu.us